tools/codegen/gen.py

```diff
@@ -814,6 +814,17 @@ def compute_declaration_yaml(f: NativeFunction) -> object:
         ('has_math_kernel', f.dispatch is not None and 'Math' in f.dispatch),
     ])
 
+@with_native_function
+def compute_registration_declarations(f: NativeFunction) -> str:
+    name = cpp.name(f.func)
```

Comment:
Not really the fault of this PR, but maybe we should add `dispatcher.name` and point it at `cpp.name`; would help make this look more uniform